arlo_server/__init__.py

```diff
@@ -964,8 +964,14 @@ def me():
         return jsonify(type=user_type,
                        email=user_email,
                        organizations=[{
-                           "id": org.id,
-                           "name": org.name
+                           "id":
+                           org.id,
+                           "name":
+                           org.name,
+                           "elections": [{
+                               "id": e.id,
+                               "name": e.name
```

Comment:
@benadida Can we also get the `created_at` value as well, since @mcchilders wants them sorted by that and have the dates displayed as well on the list?

Reply:
do we want `created_at` or do we want the stated date of the election and/or audit? We don't currently have the `created_at`.

Reply:
@benadida I added that field in my other PR for listing audits as an auto-populated field when an audit is created. Should we add that here?

https://github.com/votingworks/arlo/pull/267/files#diff-f219bfe69e6ed201e4bdfdb371dc0c9bR17